Example/QuranEngineApp/Classes/AppDelegate.swift

```diff
@@ -49,6 +49,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let downloadManager = container.downloadManager
         downloadManager.setBackgroundSessionCompletion(completionHandler)
     }
+    
+    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
+        // Check if the URL scheme matches the custom scheme
+        if url.scheme == "quran-ios" {
+            // Parse the URL to determine the desired action
+            let path = url.host ?? ""
```

Comment:
host is not the path. We should look at the url.path rather than the host.

Reply:
addressed